app/components/post_list/post/body/files/index.ts

```diff
@@ -32,11 +54,15 @@ const enhance = withObservables(['post'], ({database, post}: {post: PostModel} &
         map(([download, compliance]) => compliance || download),
     );
 
+    const filesInfo = post.files.observeWithColumns(['local_path']).pipe(
```

Comment:
Not sure what else can change, but are we sure it is enough observe the local path only?

Reply:
yes, I don't expect anything else in the files changing, in fact previously we were only listening for changes in the list, that should not happen either as posts cannot be edited to change files data.